### PR TITLE
Release/6.5.0

### PR DESCRIPTION
**Pull request by kopijunkie**
### Added

- Public: Added npm latest version badge.
- Internal: Now the UI tests will hit API endpoints from a dockerised mock server.
- Internal: Introduce TypeScript on non-critical components & deprecate FlowType.
- Internal: Introduce integration tests for API endpoint integrations.

### Changed

- Internal: Switched to `bundlewatch` Github Action for bundle size diff reporting, checking.
- UI: Updated text and background colours.
- Public: Removed references to MIT license in README and updated copy in LICENSE to refer to Onfido Service Agreement and 3rd party licenses reports.
- Public: Return error for image quality failures on the first two document upload attempts.

### Fixed

- Public: Fix "File type not supported" error on snapshot upload in selfie step.
- Public: Fix typo in `en_US.json` file

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1326-pr-onfido-sdk-ui-onfido.surge.sh

File: MIGRATION.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 The guides below are provided to ease the transition of existing applications using the Onfido SDK from one version to another that introduces breaking API changes.
 
+## `6.x.y` -> `7.0.0`
```
Comment (ethan605) [inline MIGRATION.md:5]: I forgot to update this
```suggestion
## `6.4.0` -> `6.5.0`
```

File: MIGRATION.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+
+From version 7.0.0, TypeScript is officially supported. If you previously installed
```
Comment (ethan605) [inline MIGRATION.md:7]:
```suggestion
From version 6.5.0, TypeScript is officially supported. If you previously installed
```

File: README.md [hunk 3]
```diff
@@ -797,6 +799,17 @@ The Onfido SDK has been optimised to provide the following accessibility support
 
 Refer to our [accessibility statement](https://developers.onfido.com/guide/sdk-accessibility-statement) for more details.
 
+## TypeScript
+
+From version `7.0.0`, TypeScript is officially supported, providing typings for:
```
Comment (ethan605) [inline README.md:804]:
```suggestion
From version `6.5.0`, TypeScript is officially supported, providing typings for:
```

**Review by ethan605 — CHANGES_REQUESTED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1326-pr-onfido-sdk-ui-onfido.surge.sh

**Review by ethan605 — APPROVED** [review]
LGTM!

**Review by stefaniacardenas — APPROVED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1326-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1326-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**